cmd/coordinated/metrics.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2015-2017 Diffeo, Inc.
+// This software is released under an MIT/X11 open source license.
+
+package main
+
+import (
+	"github.com/diffeo/go-coordinate/coordinate"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var coordinateSummary = prometheus.NewGaugeVec(
+	prometheus.GaugeOpts{
+		Namespace: "diffeo",
+		Subsystem: "coordinate",
+		Name:      "coordinate_summary",
+		Help:      "Summary of coordinate work specs",
```

Comment:
`"Number of Coordinate work units"`

I'm not sure if this is conventionally short (like these texts) or sentence-length or paragraph-length.